Chocolatey.Cake.Recipe/Content/packages.cake

```diff
@@ -110,6 +110,23 @@ BuildParameters.Tasks.CreateNuGetPackagesTask = Task("Create-NuGet-Packages")
     }
 });
 
+BuildParameters.Tasks.CreateTarGzPackagesTask = Task("Create-TarGz-Packages")
+    .IsDependentOn("Clean")
+    .IsDependentOn("Copy-Nuspec-Folders")
+    .WithCriteria(BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows, "Skipping because running on Windows")
+    .Does(() =>
+{
+    StartProcess(
+        "tar",
+        string.Format(
+            "-czvf {0}/chocolatey.v{1}.tar.gz {2}",
+            BuildParameters.Paths.chocolateyPackagesOutputDirectory,
```

Comment:
I don't think this is right. `chocolateyPackagesOutputDirectory` is not a public property.

Reply:
I think you want `BuildParameters.Paths.ChocolateyPackages`